hooks/post/10-post_install.rb

```diff
@@ -35,16 +35,16 @@ def new_install?
 
     if Kafo::Helpers.module_enabled?(@kafo, 'katello')
       say <<MSG
-  * To install an additional capsule on a separate machine run the following command:
+  * To install an additional Foreman proxy on separate machine continue by running:
 
-      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "<%= color('/root/$CAPSULE-certs.tar', :info) %>"
+      foreman-proxy-certs-generate --foreman-proxy-fqdn "<%= color('$FOREMAN-PROXY', :info) %>" --certs-tar "<%= color('~/$FOREMAN-PROXY-certs.tar', :info) %>"
 
 MSG
     end
 
-    if Kafo::Helpers.module_enabled?(@kafo, 'capsule_certs')
-      if certs_tar = @kafo.param('capsule_certs', 'certs_tar').value
-        capsule_fqdn          = @kafo.param('capsule_certs', 'capsule_fqdn').value
+    if Kafo::Helpers.module_enabled?(@kafo, 'foreman_proxy_certs')
+      if certs_tar = @kafo.param('foreman_proxy_certs', 'certs_tar').value
+        foreman_proxy_fqdn          = @kafo.param('foreman_proxy_certs', 'foreman_proxy_fqdn').value
```

Comment:
Nitpick, can you realign the ='s spacing to match the ones below (Or remove the alignment altogother)?